src/schema/v2/sale/index.ts

```diff
@@ -178,6 +178,12 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ cascading_end_time_interval }) =>
           cascading_end_time_interval,
       },
+      cascadingcascadingEndTimeIntervalMinutesEndTimeInterval: {
+        type: GraphQLInt,
+        description: "Amount of minutes in between each lot closing.",
+        resolve: ({ cascading_end_time_interval }) =>
+          cascading_end_time_interval / 60,
```

Comment:
This might be odd for sales that don't have this field enabled.

Since this field is nullable and we're using the presence to determine if a sale has this enabled, we should leave the type as-is (nullable `GraphQLInt`), and probably `return null` if `cascading_end_time_interval` is not set, otherwise we do the ` / 60` thing.